#437 Modified introduction page including ML (#438)
Inbibo/adonisfx_docs: docs/index.md

# Introduction

Welcome to the technical documentation for Adonis.

Adonis provides tools for building, simulating, and accelerating high-quality deformation workflows for digital characters. It is available for Maya and Houdini, with workflows that support both physics-based simulation and machine learning-assisted deformation.

The documentation is organized around two main areas:

- **AdonisFX**: Tools for creature and character simulation, including muscle, fat, skin, face, and related deformation workflows.

> - **AdonisML**: Machine learning tools for digital anatomy workflows, including data extraction, neural clustering, model training, pose-based neural deformation, and smart tissue simulation.

Reviewer: Applied

AdonisML works in two main layers:

- **AdonisML Deformer**: A pose-based neural deformer that provides interactive-speed feedback for high-quality skin deformation.
- **Smart tissue**: A dynamics layer that adds inertial detail such as jiggle, settle, and overshoot, which pose-based ML deformation does not usually capture on its own.

The AdonisML Deformer allows animators to work interactively with a high-quality representation of the character deformation. Because pose-based ML deformers are trained from pose data, their output is based on the current pose and does not directly represent motion, velocity, or dynamic history.

AdonisML provides tools to define narrow locality for ML deformation, helping the trained model isolate local deformation behavior more effectively. Neural clustering and training workflows allow users to guide how different regions of the character are learned and evaluated.

Smart tissue complements the pose-based deformer by adding dynamic behavior on top of the ML-driven result. It uses neural predictions of underlying muscle activation to drive simulation behavior without requiring artists to control complex physical parameters directly.

> Together, the AdonisML Deformer and smart tissue provide a workflow for interactive deformation feedback and fast dynamic detail. This allows teams to enhance simulation results without always requiring a full anatomy simulation setup for every iteration or shot.

Reviewer: Here we are not referring to node names (yet), so I am going to use "ML Deformer" and "Smart Tissue"

- **Quick Setup**. Preparing a character for realistic deformation can be time-consuming and often requires specialized knowledge of physical parameters, rig structure, and deformation workflows. Adonis provides intuitive tools and parameters that help artists and technical directors set up complex deformation systems more efficiently.

- **Modular Workflows**. AdonisFX components can be combined in different ways to support full simulation setups, targeted deformation setups, and hybrid simulation approaches. This allows teams to adapt the workflow to the needs of each character or shot.

- **Interactive Feedback**. AdonisML enables animators to work with interactive-speed feedback from trained neural deformation models. This helps reduce the time spent waiting for simulation or playblast results during shot iteration.

- **Local ML Deformation**. AdonisML provides workflows for defining local deformation regions, helping ML models learn and evaluate deformation with narrower locality where needed.

> - **Dynamic Detail**. Smart tissue adds dynamic behavior such as jiggle, settling, and overshoot to ML-driven deformation workflows. This helps recover motion-driven detail that pose-based neural deformation does not capture by itself.

Reviewer: yes, applied

- **Pipeline Flexibility**. Adonis workflows can be integrated into animation rigs, rigging pipelines, CFX pipelines, and procedural workflows. Adonis data can also support cross-DCC workflows between Maya and Houdini where supported.

## Why Adonis?

> With Adonis you can:

Reviewer: applied

- integrate simulation and deformation tools into animation rigs,
- integrate deformation workflows into rigging, CFX, and procedural pipelines,
- build modular physics-based character deformation setups with artist-friendly controls,
- combine components in different ways depending on the character, shot, or pipeline,

> - use hybrid simulation workflows when only specific character regions need simulation,

Reviewer 2: @carlosmonteagudoinbibo maybe we can follow the suggestion here?

> - use hybrid simulation workflows when only specific character regions need simulation,

Reviewer: applied, yes!

# Introduction

Welcome to the technical documentation for Adonis.

Adonis provides tools for building, simulating, and accelerating high-quality deformation workflows for digital characters. It is available for Maya and Houdini, with workflows that support both physics-based simulation and machine learning-assisted deformation.

The documentation is organized around two main areas:

- **AdonisFX**: Tools for creature and character simulation, including muscle, fat, skin, face, and related deformation workflows.

> - **AdonisML**: Machine learning tools for digital anatomy workflows, including data extraction, neural clustering, model training, pose-based neural deformation, and smart tissue simulation.

Reviewer 2: ```suggestion
- **AdonisML**: Machine learning tools for digital anatomy workflows, including data extraction, neural clustering and model training tools to drive pose-based neural deformation and smart tissue ml driven simulation.
```

Reviewer: Applied

If you need more information or help, please send an email to **adonis.support@inbibo.co.uk**.

## AdonisFX

**AdonisFX** provides fast and intuitive tools for creature and character deformation. It includes modular solvers and deformers for muscle, fat, skin, face, and related physical or procedural deformation workflows.

The modularity of the AdonisFX components allows artists and technical directors to set up characters in multiple ways, depending on the needs of the rig, shot, or production pipeline. Components can be combined to build full digital anatomy setups, targeted deformation setups, or hybrid simulation workflows where only selected parts of the character require simulation.

AdonisFX is designed to help teams build high-quality digital anatomy setups directly inside Maya and Houdini. These workflows can be integrated into animation rigs, rigging pipelines, CFX pipelines, and procedural character workflows.

## AdonisML

**AdonisML** brings machine learning into the Adonis digital anatomy framework.

> AdonisML works in two main layers:

Reviewer 2: ```suggestion
AdonisML can be used to further optimize the efficiency and complexity of an Adonis setup. AdonisML features mainly apply two layers:
```

Reviewer: Applied

> - **AdonisML Deformer**: A pose-based neural deformer that provides interactive-speed feedback for high-quality skin deformation.

Reviewer 2: ```suggestion
- **ML Deformer**: A pose-based neural deformer that provides interactive-speed feedback for high-quality skin deformation.
```

Reviewer: Applied

> - **Smart tissue**: A dynamics layer that adds inertial detail such as jiggle, settle, and overshoot, which pose-based ML deformation does not usually capture on its own.

Reviewer 2: ```suggestion
- **Smart tissue**: A fast and procedural simulation layer that adds inertial detail such as jiggle, settle, and overshoot, which pose-based ML deformation does not usually capture on its own. This can be further enhanced by the use of a neural network for dynamic material properties prediction.
```

Reviewer: Applied

> 

Reviewer 2: ```suggestion

> [!IMPORTANT]
> ML Deformer and Smart Tissue nodes can be used within an Adonis **FX** license bundle, however training of neural networks for ML deformation and ML driven simulation are allowed for **ML** license bundles. See more info in <bundles page>

```
Please add link to the bundles section from the licensing page.

> The AdonisML Deformer allows animators to work interactively with a high-quality representation of the character deformation. Because pose-based ML deformers are trained from pose data, their output is based on the current pose and does not directly represent motion, velocity, or dynamic history.

Reviewer 2: ```suggestion
The Adonis ML Deformer allows animators to work interactively with a high-quality representation of the character deformation. Because pose-based ML deformers are trained from pose data, their output is based on the current pose and does not directly represent motion, velocity, or dynamic history.
```

Reviewer: Replaced to "The ML Deformer"

AdonisML provides tools to define narrow locality for ML deformation, helping the trained model isolate local deformation behavior more effectively. Neural clustering and training workflows allow users to guide how different regions of the character are learned and evaluated.

> Smart tissue complements the pose-based deformer by adding dynamic behavior on top of the ML-driven result. It uses neural predictions of underlying muscle activation to drive simulation behavior without requiring artists to control complex physical parameters directly.

Reviewer 2: ```suggestion
Smart tissue complements the pose-based deformer by adding dynamic behavior on top of the ML-driven result. It can use neural predictions of underlying muscle activation to drive simulation behavior without requiring artists to control complex physical parameters directly.
```

Reviewer: yes, applied

> Together, the AdonisML Deformer and smart tissue provide a workflow for interactive deformation feedback and fast dynamic detail. This allows teams to enhance simulation results without always requiring a full anatomy simulation setup for every iteration or shot.

Reviewer 2: ```suggestion
Together, the Adonis ML Deformer and Smart Tissue provide a workflow for interactive deformation feedback and fast dynamic detail. This allows teams to enhance simulation results without always requiring a full anatomy simulation setup for every iteration or shot.
```

Reviewer: Applied, here I am using "ML Deformer" without Adonis prefix
